internal/status/crd_status.go

```diff
@@ -64,6 +67,25 @@ func UpdateHostRuleStatus(key string, hr *akov1beta1.HostRule, updateStatus Upda
 		}
 		UpdateHostRuleStatus(key, updatedHr, updateStatus, retry+1)
 	}
+	// wait for crdinformer to be updated
+	constantBackoff := backoff.WithMaxRetries(backoff.NewConstantBackOff(500*time.Millisecond), 5)
```

Comment:
This is being used in main processing path. It will impact each and every status update. Can it be optimized or moved to some other place?

Reply:
Is this being done to make sure that informer cache is populated before AKO uses hostrule in graph layer, did we see any issue related to this and if that's the case why aren't we doing the same for other CRs?

Reply:
as discussed, we will take for other CRs separately